packages/components/src/components/List/ListItem.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactNode } from "react";
+import { Box } from "../../primitives/Box";
+
+export interface ListItemProps
+  extends Omit<React.HTMLAttributes<HTMLLIElement>, "color"> {
```

Comment:
And also remove it from here.

Reply:
Same here https://github.com/Localitos/pluto/pull/147#discussion_r1006881417